src/main/java/org/prebid/server/spring/config/WebConfiguration.java

```diff
@@ -83,16 +83,19 @@ HttpServerOptions httpServerOptions(
             @Value("#{'${http.max-initial-line-length:${server.max-initial-line-length:}}'}") int maxInitialLineLength,
             @Value("#{'${http.ssl:${server.ssl:}}'}") boolean ssl,
             @Value("#{'${http.jks-path:${server.jks-path:}}'}") String jksPath,
-            @Value("#{'${http.jks-password:${server.jks-password:}}'}") String jksPassword) {
+            @Value("#{'${http.jks-password:${server.jks-password:}}'}") String jksPassword,
+            @Value("#{'${http.enable-quickack:${server.enable-quickack}}'}") boolean enableQuickAck,
+            @Value("#{'${http.enable-reuseport:${server.enable-reuseport}}'}") boolean enableReusePort) {
 
         final HttpServerOptions httpServerOptions = new HttpServerOptions()
                 .setHandle100ContinueAutomatically(true)
                 .setMaxInitialLineLength(maxInitialLineLength)
                 .setMaxHeaderSize(maxHeaderSize)
                 .setCompressionSupported(true)
                 .setDecompressionSupported(true)
-                .setIdleTimeout(10); // kick off long processing requests
-
+                .setIdleTimeout(10) // kick off long processing requests
```

Comment:
Looks like this one should be configurable

Reply:
Sure, not a problem.

Reply:
Done.